src/backend/joanie/core/models/contracts.py

```diff
@@ -170,34 +171,24 @@ def generate(self):
             raise PermissionDenied(
                 _("This contract is already signed and can not be regenerated.")
             )
-
         body = self.definition.body
-
         # Compute and save body of this definition body
         md5_body_hash = hashlib.md5(body.encode(), usedforsecurity=False)
         self.definition_checksum = md5_body_hash.hexdigest()
-        self.context = self.get_document_context()
```

Comment:
I do not understand why we do not need anymore to generate then persiste context ?